client/app/src/main/java/com/microsoft/research/karya/ui/scenarios/common/MicrotaskRenderer.kt

```diff
@@ -262,6 +294,12 @@ abstract class MicrotaskRenderer(
   final override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
 
+    // Setting up assistant
+    assistant = Assistant(this)
+
+    // Set db and API service
+    karyaDb = KaryaDatabase.getInstance(this)!!
```

Comment:
This can be injected from the DI

Reply:
This while class will change when we add Viewmodel